packages/eui/src/components/icon/icon_map.ts

```diff
@@ -79,6 +79,7 @@ export const typeToPathMap = {
   createSingleMetricJob: 'ml_create_single_metric_job',
   cross: 'cross',
   crossClusterReplicationApp: 'app_cross_cluster_replication',
+  crossInCircle: 'crossInCircle',
```

Comment:
Once the SVG file name is update, you'll also need to update this.

```suggestion
  crossInCircle: 'cross_in_circle',
```